target/linux/rtl838x/base-files/etc/board.d/02-network

```diff
@@ -0,0 +1,95 @@
+#!/bin/sh
+
+. /lib/functions.sh
+. /lib/functions/uci-defaults.sh
+. /lib/functions/system.sh
+
+rtl838x_setup_switch()
+{
+	local switchid net portid master device lan_role lan_list
+
+	json_select_object switch
+	# Find slave ports
+	for net in `ls -d /sys/class/net/*`; do
```

Comment:
Please use $() syntax instead of deprecated backticks (also below).

I'm not sure whether this kind of network setup is acceptable, though.

Reply:
Will change to $() syntax.
All of these ports are configured and named in the .dts files, there are switches with 52(!) ports. Switches of the type only have LAN ports, which are always labelled from 1 to N. In order to support a new device this auto-configuration is very useful, you avoid making mistakes and you are much quicker, think of having dozens of ports for each type of device (there are only about a 12 different types, but nevertheless) in this file, it would become unreadable. Already the huge .dts files are a nuisance. The setup is also only done once during a first boot, so it does not cost time.